### ISSUE TITLE
The Compel .and() syntax seems broken for SDXL

### ISSUE THREAD

**Issue by AvidGameFan** [easydiffusion/easydiffusion#1735]
If I use the .and() syntax with an SDXL model, I get:

**Error: All conditioning tensors must have the same batch size (1) and number of embeddings per token (77**

Switching to an SD 1.5 model, the syntax works fine.  I didn't test other Compel functions such as .blend().

(Documentation on the Compel syntax is here: https://github.com/damian0815/compel?tab=readme-ov-file#120---concatenate-embeddings-using-and )

In the below prompt, a reasonable image should be produced, not an error.
```
("taylor swift wearing blue jeans","emma watson wearing shorts").and()
```

Desktop environment:
 - OS: Windows
 - Browser: Edge
 - Version: Current non-beta

**Comment by cmdr2** [comment]
Agreed, but sorry this is unlikely to receive a fix since ED's moving to Forge in v3.5.
